phpunit.xml.dist

```diff
@@ -45,6 +45,7 @@
             <exclude>src/Backend/Modules/Groups/Tests/Actions</exclude>
             <exclude>src/Backend/Modules/Locale/Tests/Actions</exclude>
             <exclude>src/Backend/Modules/Location/Tests/Actions</exclude>
+            <exclude>src/Backend/Modules/Pages/Tests/Actions</exclude>
```

Comment:
is it still not possible in php unit to use excludes with a regex pattern? or the `**` ?